store/store_download.go

```diff
@@ -977,3 +977,7 @@ func (s *Store) SetCacheDownloads(fileCount int) {
 		s.cacher = &nullCache{}
 	}
 }
+
+func (s *Store) CleanDownloadsCache() error {
```

Comment:
this needs a doc comment, probably a good time as any to add also for the other Cache* methods here